public/styles/components/_link.scss

```diff
@@ -77,12 +91,5 @@ a {
         }
       }
     }
-
-    &.bsds-link-mailto {
-      &::after {
-        -webkit-mask: url('../../assets/images/icon-envelope.svg') no-repeat 50% 50%;
-        mask: url('../../assets/images/icon-envelope.svg') no-repeat 50% 50%;
-      }
-    }
```

Comment:
I'm not sure how to get the "icon-arrow-down" icon out of [Zeplin](https://app.zeplin.io/project/6075edaf528a0a46f9447bd5/screen/64513eff73de712645877472) and into the codebase as an SVG. The transform yields a similar result, but please help if you'd rather import a new SVG 😀

See the Link.tsx file below for more on this classname change.

![Screenshot 2023-11-08 at 12 22 44 PM](https://github.com/bos-sci/anatomy-react/assets/96078669/312a2c06-bb7b-4316-8b9f-110932f27ae5)